packages/app/obojobo-express/server/routes/api/visits.js

```diff
@@ -91,6 +99,26 @@ router
 					visitId: req.currentVisit.id
 				})
 			})
+			.then(() => viewerUsersState.getLastLogin(userId))
+			.then(result => {
+				if (result) lastLogin = result.last_login
+			})
```

Comment:
Ideally req.currentUser would just have a lastLogin property that you could check, rather than creating a whole new function to get that information separately.